packages/menu/test/menu-item.test.ts

```diff
@@ -169,6 +169,23 @@ describe('Menu item', () => {
         expect(el.value).to.equal('Selected Text');
         expect(el.hasAttribute('value')).to.be.false;
     });
+    it('assigns content to the description slot', async () => {
+        const el = await fixture<MenuItem>(
+            html`
+                <sp-menu-item selected>
+                    Menu Item Text
+                    <span slot="description">
+                        Description for the Menu-Item
+                    </span>
+                </sp-menu-item>
+            `
+        );
+
+        const descriptionElement = el.shadowRoot.querySelector(
+            'slot[name="description"]'
+        );
+        expect(descriptionElement).to.not.be.null;
```

Comment:
Agree with Westbrook that this should be updated to `descriptionElement.assignedSlot` as he noted.